src/main/java/net/goldiriath/plugin/game/inventory/InventoryUtil.java

```diff
@@ -83,4 +117,52 @@ public static boolean storeInInventory(PlayerInventory inv, ItemStack stack) {
 
     }
 
+    public static WeaponType getWeaponType(Player player) {
+        // Returns the WeaponType that is in the first slot of the players hotbar
+        ItemStack stack = getWeapon(player);
+        if(stack == null) {
+            return null;
+        }
+
+        switch (stack.getType()) {
+            case WOOD_SWORD:
+            case STONE_SWORD:
+            case GOLD_SWORD:
+            case IRON_SWORD:
+            case DIAMOND_SWORD:
+                return WeaponType.SWORD;
+            case BOW:
+                return WeaponType.BOW;
+            case EMERALD:
+                return WeaponType.WAND;
+            case SHEARS:
+                return WeaponType.KNIFE;
+        }
+        return null;
+    }
+
+    public static ItemStack getWeapon(Player player) {
+        // Returns the ItemStack that is in the first slot of the players hotbar
+        return player.getInventory().getItem(0);
+    }
+
+    /**
+     * method returns the position of the first similar item found in the PlayerInventory.
```

Comment:
Needs better Javadoc formatting. For examples, see [this link](http://www.oracle.com/technetwork/articles/java/index-137868.html) or [this link](https://hub.spigotmc.org/stash/projects/SPIGOT/repos/bukkit/browse/src/main/java/org/bukkit/Bukkit.java#191-199) (containing similar methods).